### ISSUE TITLE
NBC Nightly News

### ISSUE THREAD

**Issue by ryandesign** [rg3/youtube-dl#4884]
Hello,

The [supported sites page](http://rg3.github.io/youtube-dl/supportedsites.html) lists "NBCNews", however I am unable to download the NBC Nightly News, for example:

```
$ youtube-dl --version
2015.02.04
$ youtube-dl http://www.nbcnews.com/nightly-news/video/nightly-news-with-brian-williams-full-broadcast-february-4-394064451844
[generic] nightly-news-with-brian-williams-full-broadcast-february-4-394064451844: Requesting header
WARNING: Falling back on generic information extractor.
[generic] nightly-news-with-brian-williams-full-broadcast-february-4-394064451844: Downloading webpage
[generic] nightly-news-with-brian-williams-full-broadcast-february-4-394064451844: Extracting information
ERROR: Unsupported URL: http://www.nbcnews.com/nightly-news/video/nightly-news-with-brian-williams-full-broadcast-february-4-394064451844
```

NBC has decided to discontinue publishing their NBC Nighty News video podcast, so I am trying to use youtube-dl to build a new personal viewing solution for this content. Thanks for your help.

**Comment by robin007bond** [comment]
I think that it's a problem with the _VALID_URL regex in NBCNewsIE:

```
_VALID_URL = r'''(?x)https?://www\.nbcnews\.com/
        ((video/.+?/(?P<id>\d+))|
        (feature/[^/]+/(?P<title>.+)))
        '''
```

**Comment by dstftw** [comment]
NBC nightly news will be supported in the next version, thanks for the report.

**Comment by phihag** [comment]
NBC support has been included in youtube-dl 2015.02.16. See [our FAQ](https://yt-dl.org/update) if you need help updating.
